ambry-rest/src/main/java/com.github.ambry.rest/NettyMetrics.java

```diff
@@ -237,5 +247,24 @@ public NettyMetrics(MetricRegistry metricRegistry) {
     // NettyServer
     nettyServerShutdownTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "ShutdownTimeInMs"));
     nettyServerStartTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "StartTimeInMs"));
+    // ConnectionStatsHandler
+    connectionsConnectedCount =
+        metricRegistry.counter(MetricRegistry.name(NettyServer.class, "ConnectionsConnectedCount"));
+    connectionsDisconnectedCount =
+        metricRegistry.counter(MetricRegistry.name(NettyServer.class, "ConnectionsDisconnectedCount"));
+  }
+
+  /**
+   * Register {@link ConnectionStatsHandler#openConnections} to track open connections
```

Comment:
`openConnections` is a private variable. Don't make it a part of a `public` (or package private) function's doc.